examples/inlinebot.py

```diff
@@ -48,31 +50,29 @@ def escape_markdown(text):
 
 
 def inlinequery(bot, update):
+    """Handle the inline query."""
     query = update.inline_query.query
-    results = list()
-
-    results.append(InlineQueryResultArticle(id=uuid4(),
-                                            title="Caps",
-                                            input_message_content=InputTextMessageContent(
-                                                query.upper())))
-
-    results.append(InlineQueryResultArticle(id=uuid4(),
-                                            title="Bold",
-                                            input_message_content=InputTextMessageContent(
-                                                "*%s*" % escape_markdown(query),
-                                                parse_mode=ParseMode.MARKDOWN)))
-
-    results.append(InlineQueryResultArticle(id=uuid4(),
-                                            title="Italic",
-                                            input_message_content=InputTextMessageContent(
-                                                "_%s_" % escape_markdown(query),
-                                                parse_mode=ParseMode.MARKDOWN)))
+    results = [
+               InlineQueryResultArticle(id=uuid4(),
+                                        title="Caps",
+                                        input_message_content=InputTextMessageContent(
+                                                query.upper())),
+               InlineQueryResultArticle(id=uuid4(),
+                                        title="Bold",
+                                        input_message_content=InputTextMessageContent(
+                                                "*{}*".format(escape_markdown(query)),
+                                                parse_mode=ParseMode.MARKDOWN)),
+               InlineQueryResultArticle(id=uuid4(),
+                                        title="Italic",
+                                        input_message_content=InputTextMessageContent(
+                                                "_{}_".format(escape_markdown(query)),
+                                                parse_mode=ParseMode.MARKDOWN))]
```

Comment:
why did you change the indentation here? please revert.

Reply:
I changed the entire code block here. Before my changes the Items were appended to the list, now the list is created with those items.